js/main.js

```diff
@@ -40,6 +48,37 @@ Site = {
 
 };
 
+Site.Shop = {
+  init: function() {
+    var _this = this;
+
+    this.Search.init();
+  },
+
+  Search: {
+    init: function() {
+      var _this = this;
+
+      if ($('#searchform').length) {
+        _this.bindFocus();
+      }
+    },
+
+    bindFocus: function() {
```

Comment:
its for styling. if you look at design you'll see that before focus, all there is is a magnifying glass, so i would assume that users would click on that thinking it would activate the search, but actually that magnifying glass is also the submit button, so this way it focuses the text input on first click rather than submitting